### ISSUE TITLE
PostgreSQLQuery only supports updating one field and only one record when use 'ON CONFLICT'

### ISSUE THREAD

**Issue by s-khaon** [kayak/pypika#363]
```
    @classmethod
    def _build_insert_sql(cls, data):
        """
        :param data:
        :return:
        """
        assert cls.insert_table_name and cls.insert_fields
        table = cls.insert_table_name
        insert_fields = cls.insert_fields
        update_fields = cls.update_fields
        conflict = cls.insert_conflict
        returning = cls.returning
        is_constraint = cls.is_constraint
        q = PostgreSQLQuery.into(table).columns(*insert_fields).insert(*data)
        sql = str(q)
        if conflict:
            if is_constraint:
                sql += ' ON CONFLICT ON CONSTRAINT {}'.format(conflict)
            else:
                sql += ' ON CONFLICT ({})'.format(conflict)
            if update_fields:
                update_str = ",".join(map(lambda x: '"{field}"=EXCLUDED.{field}'.format(field=x), update_fields))
                sql += " DO UPDATE SET {}".format(update_str)
            else:
                sql += " DO NOTHING"
        if returning:
            sql += ' RETURNING "{}"'.format(returning)
        logger.debug(cls.__name__ + ": %s" % sql[:512])
        return sql
```

**Comment by taybin** [comment]
See also #332

**Comment by th0rgall** [comment]
If I understand the issue right, I think it is (now?) not an issue anymore. 

I bumped into this because I also needed to update multiple fields in `.on_update()`, and I couldn't find pointers on how to do this in the readme documentation or API reference. But after some searching in the codebase, it turns out that just like you can chain multiple `.insert()` calls to insert multiple values, you can also chain multiple `.do_update()` calls to update multiple fields.

This test shows an example: https://github.com/kayak/pypika/blob/53a77eb9b5bda70fd0780b66a4f68daa14fe3682/pypika/tests/test_inserts.py#L310-L330

Unlike `.insert()`, it doesn't look like you can pass multiple field (& value) tuples to one `.do_update()` call though, I've tried.

**Comment by s-khaon** [comment]
fixed at  #374[https://github.com/kayak/pypika/issues/374](url)
